docs/developer-docs/multi-chain/overview.mdx

```diff
@@ -0,0 +1,115 @@
+
+Integrations with different chains vary in how they utilize these features to facilitate the integration.
+
+### Bitcoin integration
+
+The integration with Bitcoin primarily uses the threshold ECDSA feature along with a Bitcoin adapter. Threshold ECDSA is used to securely store a smart contract's secret key, which is used to sign transactions. Bitcoin supports ECDSA, allowing ICP smart contracts to directly create and sign transactions on the Bitcoin network. Canisters can also create a Bitcoin address, then send and receive bitcoin as if they were a traditional Bitcoin user.
+
+The Bitcoin adapter is a subnet of nodes that participate in the Bitcoin network. They fetch the Bitcoin network state and blocks, then facilitate sending transactions to the Bitcoin network.
+
+[Learn more about how the Bitcoin integration works](/docs/current/developer-docs/multi-chain/bitcoin/overview).
+
+### Ethereum integration
+
+The integration with Ethereum also uses threshold ECDSA for canisters to create and sign transactions directly on the Ethereum network. However, instead of an adapter like the Bitcoin integration uses, the Ethereum integration uses a full-node API that utilizes HTTPS outcalls to query information from the Ethereum network and submit transactions. This API can be used through the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc). The EVM RPC canister supports several RPC services, such as CloudFlare, Ankr, BlockPI, and Alchemy.
+
+[Learn more about how the Ethereum integration works](/docs/current/developer-docs/multi-chain/ethereum/overview).
+
+## Benefits of multi-chain
+
+When compared to traditional blockchain bridges, ICP's multi-chain capabilities have several benefits:
+
+- **Enhanced programmability**: Dapps can expand their functionalities and capabilities beyond just ICP, allowing for new use-cases and applications.
+
+- **Orchestration**: Transactions on integrated networks can be orchestrated and automated using ICP smart contracts.
+
+- **Security and privacy**: ICP's direct integration with other chains is more secure than using an intermediary blockchain bridge, which may have security vulnerabilities and be exploited.
+
+- **Lower transaction costs**: Submitting transactions to Bitcoin or Ethereum from an ICP smart contract have a significantly lower cost than submitting transactions on the chains directly.
+
+- **Faster transaction times**: Transaction times on Bitcoin and Ethereum are drastically reduced by using the ICP multi-chain integrations for each.
+
+## Multi-chain integrations
+
+Current multi-chain integrations include Bitcoin and Ethereum, and the EVM RPC canister can be used to interact with any EVM-compatible chain, such as Polygon or Avalanche.  Direct integration with other chains is possible, and developers can build their own.
```

Comment:
IS there a special reason for mentioning Polygon? Can we instead refer to the most decentralized L2?
```suggestion
Current multi-chain integrations include Bitcoin and Ethereum, and the EVM RPC canister can be used to interact with any EVM-compatible chain, such as Arbitrum or Avalanche.  Direct integration with other chains is possible, and developers can build their own.
```